cfme/web_ui/flash.py

```diff
@@ -179,6 +179,16 @@ def assert_message_contain(m):
 
 @verify_rails_error
 @verpick_message
+def message_contain(m):
```

Comment:
Why create a new function for this? Just wrap the assert_message_contain inside a ```try: except:``` block